src/pages/docs/infrastructure/workers/autoscaling-workers/horizontal-scaling.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Scaling Behavior
+description: How the worker scales
+navOrder: 30
+---
+
+The Kubernetes worker is responsible for executing work-packages received from an OctopusServer.
+
+The work-package comprises:
+* Binary packages (optional)
+* Configuration data
+* Execution script (user-defined operations which uses the config and packages supplied)
+
+The worker writes the received data to the shared file system (by default NFS), then spawns a Kubernetes pod to execute
+the supplied script. We creatively refer to these execution-pods as "script pods".
+
+For each work-package, a new script-pod is created - each operation executes in its own script-pod.
+
+## Kubernetes Horizontal Scaling
+### Default
+In a Kubernetes cluster, a pod creation request includes the resources (cpu/memory) the pod requires.
+If sufficient resources are  not available, the pod is created in the "Pending" state.
+When  required resources are available, the pod is started.
+
+Resources become available through 2 methods:
+1. Pod termination, returning the allocated resources to the cluster
+2. More nodes are added to the cluster
+
+Kubernetes decides when to start a pod based on resource _requests_, not usage.
+Meaning, if a Pod uses more resources than it requested, other pods may be starved of resources.
+
+When using the inbuilt horizontal-scaler it is important that a pod's requested resources roughly match expected usage. 
+
+## How a Kubernetes worker requests resources
+A script-pod requests a default of 2.5% of a CPU core and 100 megabytes of memory (`.Values.scriptPods.resources.requests`).
+
+As such, a kubernetes cluster will (attempt to) provision a new node when **40** script-pods are running simultaneously.
+For most cases this will result in a usable system, but it is dependent on the work being performed, and the capabilities
+of the nodes.
+
+If the cluster hosting your workers exhibits high CPU load, increasing the script-pod's requested resources may result
+in better performance.
+
+### Advanced Options
```

Comment:
Is it worth mentioning that the CPU/Memory requests and limits can be customised via the Helm chart? 
You can link to the Kubernetes docs https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ and say that we provide an 'escape hatch' in the Agent values that allows custom request/limit sizes to be set https://github.com/OctopusDeploy/helm-charts/blob/main/charts/kubernetes-agent/values.yaml#L139
Note that this works because the whole resource section gets serialised and passed to the pod creator, but you can just mention to either overwrite the `values.yaml` or set them using `--set`.

Reply:
It might also be good to put this in a more prominent section heading e.g. titled "Requests and Limits for the Worker pods" just so people who run into resource management issues can easily find this info via search.